.github/workflows/trigger_robottelo_workflow.yml

```diff
@@ -0,0 +1,20 @@
+name: Send trigger for updating robottelo image on quay.
+on:
+  push:
+    branches:
+      - master
+      - 6.*.z
+
+jobs:
+  trigger-robottelo-workflow:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger workflow in robottelo repo
+        run: |
+          curl -L \
```

Comment:
You could also use readymade Github Action instead of a curl command:

https://github.com/peter-evans/repository-dispatch 

Not a blocker comment!

Reply:
I think we don't need to use github action as we don't have much code here anyway.
but I'll check it out later and evaluate it again.